darts/models/forecasting/block_rnn_model.py

```diff
@@ -430,11 +449,31 @@ def encode_year(idx):
                     logger=logger,
                 )
 
+        # check we got right activation function specified:
+        if activation is not None:
+            try:
+                getattr(nn, activation)
+            except AttributeError:
+                raise_log(
+                    ValueError(
+                        f"Invalid activation function: {activation}. "
+                        "Please use a valid torch.nn activation function name."
+                    ),
+                    logger=logger,
+                )
+            raise_if(
+                activation and (hidden_fc_sizes is None or len(hidden_fc_sizes) == 0),
+                "The activation function has been set, but the model contains only one linear layer. "
+                "Activation function will not be applied.",
```

Comment:
Since this error interrupt the code, I would change the last sentence to make it actionable
```suggestion
                "The activation function has been set, but the model does not contain hidden fully connected layers; "
                "either set `activation=None` or increase `hidden_fc_sizes`.",
```